cylc/flow/cfgspec/globalcfg.py

```diff
@@ -826,16 +826,54 @@ def default_for(
                    range.
             ''')
             Conf('condemned', VDR.V_ABSOLUTE_HOST_LIST, desc=f'''
-                These hosts will not be used to run jobs.
+                List run hosts that workflows should *not* run on.
 
-                If workflows are already running on
-                condemned hosts, Cylc will shut them down and
-                restart them on different hosts.
+                These will be subtracted from the
+                `available <global.cylc[scheduler][run hosts]>`:
+
+                * Workflows will not start on condemned hosts.
+                * Workflows that are running on condemned hosts will attempt
+                  to migrate to an available host (providing the
```

Comment:
```suggestion
                * By default, workflows running on condemned hosts will attempt
                  to migrate to an available host (providing the
```

Reply:
Superfluous in combo with next paragraph.